### ISSUE TITLE
Unrealistic RIR ?

### ISSUE THREAD

**Issue by rfalcon100** [LCAV/pyroomacoustics#127]
I am trying to generate some basic RIR with 1 source and 1 mic,  in a 3d shoebox room. However, the resulting RIR sounds weird,. There is a chirp that goes from low to high frequencies, instead of just a decaying noise.  I am using the code provided in the examples, and have tried using different values for `fs` and `max_order`, with no success. Am I doing something wrong? Or is this the expected result?

My code:
```python
import numpy as np
import matplotlib.pyplot as plt
import sounddevice as sd
from scipy.io import wavfile

import pyroomacoustics as pra

fs = 16000
# room dimension
room_dim = [5, 4, 6]

# Create the shoebox
room = pra.ShoeBox(
    room_dim,
    absorption=0.05,
    fs=fs,
    max_order=100,
    )

# source and mic locations
room.add_source([2, 3.1, 2])
room.add_microphone_array(
        pra.MicrophoneArray(
            np.array([[2, 1.5, 2]]).T,
            room.fs)
        )

# Run the image source model
room.image_source_model()

# Plot the result up to fourth order images
room.plot(img_order=4)

plt.figure()
room.plot_rir()
plt.show()
#sd.play(room.rir[0][0],room.fs)

my_rir = room.rir[0][0] / np.max(np.abs(room.rir[0][0]))
sd.play(my_rir, fs)
wavfile.write("my_rir.wav", fs, my_rir)
```

The RIR plot:
https://postimg.cc/7Ccr5dq2

The RIR soundclip:
https://vocaroo.com/i/s0S5j9PCsB22

**Comment by rfalcon100** [comment]
I guess the result is common for the image-source method. The convolution sounds ok, but not quite what I am looking for. Still, it is a good starting point.
Thanks for the info.

**Comment by fakufaku** [comment]
Thank you for the feedback. If I may ask, what were you looking for that is missing ?

**Comment by rfalcon100** [comment]
Well, I have just started working with RIRs and I am mostly looking for simulated RIRs to expand a collection of real RIRs that I am working with. However, the simulated ones sound quite different than the real ones, even after convolving with some signal.

Maybe I need to go for a more involved method if I need more realistic sounding RIRs. Or perhaps I can adapt my scope and work with image-source.

**Comment by leumaslanda** [comment]
I also think, that RIRs are unrealistic. In my opinion, that situations is due to assumptions of the model.

Firstly, image source method is adapted from optics, where we deal with waves of significantly smaller wavelengths in comparison to obstacles they interact with. It does not consider such phenomena like diffraction (important in low frequencies), scattering (for late reflections in reverberation tail).

Moreover, the fact that absorption coefficient is frequency-dependent is ignored. Considering frequency dependent absorption coefficient should make RIR more "noisy" than "spiky".

To sum up, issues that I have mentioned can cause whether RIR will sound natural or peculiar. If you are looking for an free and open-source program generating RIR that sounds natural I will recommend you [Wayverb](https://reuk.github.io/wayverb/). Unfortunately, it is only for MACs so I've only read [implementation details](https://reuk.github.io/wayverb/evaluation.html). I think Pyroomacoustics is made generally for microphone arrays' algorithms prototyping and testing, where precise delays are far more important than modeling acoustic phenomena as thoroughly as possible.
